cabal.project

```diff
@@ -7,6 +7,7 @@ packages:
   cardano-crypto-praos
   cardano-crypto-tests
```

Comment:
Unrelated to this PR, so not a question to you, but why do some have `cardano-` in the name and other not?

Reply:
I don't know. One thing I tried to do was make all the modules I added to use a consistent naming convention.
 Cardano.Data.StrictMaybe
 Cardano.Data.StrictSeq
 Cardano.Data.StrictFingerTree
 Cardano.Data.StrictUnit